connection.js

```diff
@@ -313,7 +309,7 @@ class Connection {
             return { reject: true, msg: "Currently, " + Object.keys(this.connected_games).length + " games are being played by this bot, maximum is " + config.maxconnectedgames + " (if you see this message and you dont see any game on the bot profile page, it is because private game(s) are being played) , try again later " };
         }
 
-        const resultRank = minmaxHandicapRankActualReject("rank", user.ranking, notification.handicap, notification.ranked);
+        const resultRank = minmaxHandicapRankActualReject("rank", notification.user.ranking, notification.handicap, notification.ranked);
```

Comment:
This seems to be the online line in the entire function that uses something other than `notification.user`, If you move `minmaxHandicapRankActualReject` outside this function, you can replace the `notification` parameter with the `user` paramter. So `checkUser(notification)` -> `checkUser(user)` and you then call `checkUser(notification.user)`.

Reply:
opted for a different idea in that approaches this problem more globally, in : https://github.com/online-go/gtp2ogs/pull/209/commits/d3365737df82a1255ba6e7fb5366e42c6330165a
thanks :+1: